app/kumactl/data/install/k8s/tracing/jaeger/all-in-one-template.yaml

```diff
@@ -0,0 +1,163 @@
+#
```

Comment:
Can you maybe also add a comment with a link on which this file is based?

Reply:
Done.